app/models/miq_web_service_worker/runner.rb

```diff
@@ -1,3 +1,8 @@
 class MiqWebServiceWorker::Runner < MiqWorker::Runner
   include MiqWebServerRunnerMixin
+
+  def prepare
+    super
+    MiqApache::Control.start if MiqEnvironment::Command.is_podified?
```

Comment:
I thought we were going to avoid this and try to get it running in the pod itself always

Reply:
I can investigate that.  This is a copy of what the UI pod currently does.  I didn't know if we wanted to bring that into the scope of the SSL effort.

Reply:
yeah that's fair...I mainly reacted to an added line with `MiqApache::Control` when we are also trying to get rid of that.

Reply:
I was just thinking, is it as simple as a straight apache start right in the container boot script?